client/src/components/Calendar.js

```diff
@@ -1,49 +1,52 @@
 import React, { createRef, useState } from 'react';
-import FullCalendar from "@fullcalendar/react";
-import dayGridPlugin from "@fullcalendar/daygrid";
-import timeGridPlugin from "@fullcalendar/timegrid";
-import interactionPlugin from "@fullcalendar/interaction";
-import "@fullcalendar/core/main.css";
-import "@fullcalendar/daygrid/main.css";
-import "@fullcalendar/timegrid/main.css";
-import "../css/Calendar.css";
+import { useHistory } from 'react-router-dom';
+import FullCalendar from '@fullcalendar/react';
+import dayGridPlugin from '@fullcalendar/daygrid';
+import timeGridPlugin from '@fullcalendar/timegrid';
+import interactionPlugin from '@fullcalendar/interaction';
+import '@fullcalendar/core/main.css';
+import '@fullcalendar/daygrid/main.css';
+import '@fullcalendar/timegrid/main.css';
+import '../css/Calendar.css';
 
 
-const AppCalendar = () => { 
-
+const AppCalendar = () => {
   // state emulation, store all events user created
   const [calendarEvents, setCalendarEvents] = useState([])
 
+  const history = useHistory();
+
   // create new event function
-  const handleCreateEventClick = (arg) => {
-    let arrayObject = {
-      title: 'new event',
-      start: arg.date,
-      allDay: arg.allDay
+  const handleCreateEventClick = args => {
+    let todaysDate = new Date().toISOString().slice(0, 10);
+    if (args.dateStr === todaysDate) {
+      history.push('/symptomsurvey');
```

Comment:
I've never seen `useHistory()` in action before, but I'm certainly not a React pro regarding hooks/routes/etc... it's great to see this in action and all looks good to me!